### ISSUE TITLE
Rule proposal: disallow children prop

### ISSUE THREAD

**Issue by lencioni** [yannickcr/eslint-plugin-react#720]
Children should always be actual children, not passed in as a prop.

Bad:

```jsx
<div children='Children' />
```

Good:

```jsx
<div>
  Children
</div>
```

This should also apply to `React.createElement`.

Name: `no-children-prop`?
